egs/swbd/s5c/local/chain/tuning/run_tdnn_lstm_1e_gpudec.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# configs for 'chain' gpu decoding
+# NOTICE: 1. we need CUDA9.0 installed with correct driver version
+#         2. a GPU newer than K20
```

Comment:
What is wrong with K20?

Reply:
I use a 64 bit atomicMin operation, which is not supported earlier than k20. I change it to:
2. a GPU not earlier than K20